Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
```

Comment:
- [ ] Please make sure that the login and signup forms look exactly like the design the input file sizes and the useless text around the signup form you can use place holder instead as in the design.
![image](https://user-images.githubusercontent.com/74744787/216181847-876d307f-0613-4c05-923e-8c2bd45833db.png)